packages/design-system/src/components/N8nNotice/Notice.vue

```diff
@@ -105,16 +164,21 @@ export default Vue.extend({
 	display: flex;
 	color: var(--custom-font-black);
 	margin: 0;
-	padding: var(--spacing-xs);
+	padding: var(--spacing-2xs);
 	background-color: var(--background-color);
 	border-width: 1px 1px 1px 7px;
 	border-style: solid;
 	border-color: var(--border-color);
 	border-radius: var(--border-radius-small);
+	line-height: var(--font-line-height-compact);
 
 	a {
 		font-weight: var(--font-weight-bold);
 	}
+
+	span {
+		line-height: var(--font-line-height-compact);
+	}
```

Comment:
Otherwise overridden by text component.

Reply:
hmm could simply get rid of text component instead of fighting with it... if the default sizes don't fit

Reply:
or else add a line height prop...

Reply:
There was already a `compact` prop for Text.